### PR TITLE
Remove revive_offers_for_new_apps and revive_offers_repetitions command-line flags

### PR DESCRIPTION
**Pull request by timcharper**
The options are still accepted for now, they just don't do anything

JIRA Issues: MARATHON-8662

### CODE REVIEW

File: docs/docs/command-line-flags.md [hunk 1]
```diff
@@ -247,25 +247,14 @@ Marathon can request all available offers from Mesos again -- even those that it
 calling the underlying `reviveOffers` API call to often, you can configure the minimal delay between subsequent
 invocations of this call.
 
-* <span class="label label-default">v0.11.0</span> `--min_revive_offers_interval` (Optional. Default: 5000):
+* <span class="label label-default">v0.11.0</span> `--min_revive_offers_interval` (Optional. Default: 30000):
```
Comment (ANeumann82) [inline docs/docs/command-line-flags.md]: I assume this is the interval that you've changed back to 5 seconds in the other PR?

File: src/main/scala/mesosphere/marathon/core/launchqueue/ReviveOffersConfig.scala [hunk 2]
```diff
@@ -7,27 +7,36 @@ trait ReviveOffersConfig extends ScallopConf {
...
+      hidden = true,
+      default = Some(true),
+      prefix = "disable_")
+  }
 
   lazy val minReviveOffersInterval = opt[Long](
     "min_revive_offers_interval",
     descr = "Do not ask for all offers (also already seen ones) more often than this interval (ms).",
-    default = Some(30000))
+    default = Some(5000))
```
Comment (ANeumann82) [inline src/main/scala/mesosphere/marathon/core/launchqueue/ReviveOffersConfig.scala]: This contradicts the number from the documentation above

File: docs/docs/command-line-flags.md (hunk 1 above)
Comment (timcharper) [inline docs/docs/command-line-flags.md]:
```suggestion
* <span class="label label-default">v0.11.0</span> `--min_revive_offers_interval` (Optional. Default: 5000):
```

Comment (timcharper) [inline docs/docs/command-line-flags.md]:
```suggestion
* <span class="label label-default">v0.11.0</span> `--min_revive_offers_interval` (Optional. Default: 5000):
```

Comment (timcharper) [reply docs/docs/command-line-flags.md]: yep :)

**Pull request merged**